CONTRIBUTING.rst

```diff
@@ -234,6 +234,7 @@ changes any given branch is introducing before looking at the code.
    routines
 -  ``testing/``: improvements or changes to testing
 -  ``release/``: releases (see below)
+-  ``breaking-change/``: Changes to destroy backward compatibility
```

Comment:
While correct, I find destroy a bit strong.

```suggestion
-  ``breaking-change/``: Changes that break backward compatibility
```

Reply:
Hopefully we're not destroying backwards compatibility :)